staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -310,7 +311,7 @@ func (s *DelegatingAuthenticationOptions) ApplyTo(authenticationInfo *server.Aut
 	}
 
 	// create authenticator
-	authenticator, securityDefinitions, err := cfg.New()
+	authenticator, securityDefinitions, err := cfg.New(context.TODO())
```

Comment:
The only way I could think to wire it through here would be to make all the `ApplyTo` funcs take a context as the first arg...

Reply:
Since kubelet already has a stop channel it makes sense to hook it up, but I'm not sure it's worth the effort to make apiserver shut this down. Is this breaking the integration tests?

Reply:
I mostly just wanted to see how bad this was to wire up all the way.  I do not think it is causing any issues - I just feel uneasy when I see infinite go routines.

Reply:
Personally, I am torn between an infatuation with "crash-only software" and wanting my integration tests to not interfere with each other :)